src/pseudopeople/utilities.py

```diff
@@ -173,3 +178,16 @@ def load_ocr_errors_dict():
     )
 
     return ocr_error_dict
+
+
+def load_phonetic_errors_dict():
+    phonetic_errors = pd.read_csv(
+        paths.PHONETIC_ERRORS_DATA,
+        skiprows=[0, 1],
+        header=None,
+        names=["where", "orig", "new", "pre", "post", "pattern", "start"],
+    )
+    phonetic_error_dict = phonetic_errors.groupby("orig")["new"].apply(
+        lambda x: list(x.str.replace("@", ""))
```

Comment:
What is this doing?

Reply:
@albrja was this @ replacement in the original RT implementation? I don't really get why this is happening (but I admittedly don't know how the csv is generated in the first place)

Reply:
Yes this is just RT code (slightly change to use lambda instead of a loop).